sdf.md

```diff
@@ -1153,22 +1170,13 @@ consumption there is no conflict with the intended goal.
 ~~~
 {: #exa-sdfref}
 
-## sdfThing
-
-An sdfThing is a set of declarations and qualities that may be part of a more complex model. For example, the object declarations that make up the definition of a single socket of an outlet strip could be encapsulated in an sdfThing, and the socket-thing itself could be used in a declaration in the sdfThing definition for the outlet strip.
-
-sdfThing definitions carry semantic meaning, such as a defined refrigerator compartment and a defined freezer compartment, making up a combination refrigerator-freezer product.
+## sdfObject
 
-An sdfThing may be composed of sdfObjects and other sdfThings.
+An sdfObject is equivalent to an sdfThing with the exception and is included to ensure backwards compatability with ealier
```

Comment:
I accidentally left this in. I guess there should be no difference between `sdfObject` and `sdfthing`?

```suggestion
An sdfObject is equivalent to an sdfThing and is included to ensure backwards compatability with ealier
```

Reply:
Not just backwards compatibility, but also as a bit of expression what the modules are.

Reply:
Oh, I see. So the two terms are also going to co-exist in the future? In that case I would remove mentions of deprecation.

Reply:
Exactly.